spec/features/forms_spec.rb

```diff
@@ -31,6 +38,21 @@
       expect(page).to have_field 'Journal/Book Title', with: 'Phenomenology of perception /'
       expect(page).to have_field 'ISBN/ISSN', with: '0415834333'
     end
+    scenario 'upon submission, sends and email and renders confirmation' do
+      visit form_path({ id: 'ill', requesttype: 'ScanDelivery' }.merge(book_params))
+      # fill in required fields even though they are only required by JS
+      test_email = 'test@upenn.edu'
+      fill_in 'Email', with: test_email
+      fill_in 'Article/Chapter Title', with: 'Other Selves and the Human World'
+      fill_in 'Issue Date', with: '1945'
+      click_on 'Submit Request'
+      expect(ActionMailer::Base.deliveries.length).to eq 1
```

Comment:
Is this cleared out between tests?

Reply:
good point. i'll investigate. if not i can reset in the `before` block above

Reply:
I can point you to what I did in another project. I actually added in a before hook in `/spec_helper.rb`.

Reply:
https://github.com/cul/ac-academiccommons/blob/master/spec/spec_helper.rb#L125-L128

Reply:
thanks, i stole your code with 731298aae8b9ea4afdb1b83e9cc81a337910991b